keystore/keystore.go

```diff
@@ -1 +1,83 @@
 package keystore
+
+import (
+	"io/ioutil"
+	"log"
+
+	"github.com/gallactic/gallactic/common"
+	"github.com/gallactic/gallactic/crypto"
+	"github.com/gallactic/gallactic/keystore/key"
+)
+
+type KeyData struct {
+	Filename string
+	Label    string
+	Address  crypto.Address
+	Key      *key.Key
+}
+
+type Keystore struct {
+	Path string
+	Keys []*KeyData
+}
+
+func Open(path string) *Keystore {
+	ks := new(Keystore)
+	ks.Path = path
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	for _, file := range files {
+		fname := path + file.Name()
+		addr, lbl, v := key.VerifyFile(fname)
+
+		if v == true {
+			kd := &KeyData{fname, lbl, addr, nil}
+			ks.Keys = append(ks.Keys, kd)
+		}
+	}
+
+	return ks
+}
+
+func (ks *Keystore) New(label, auth string, valAddr bool) (*KeyData, error) {
```

Comment:
To make compatible with [key.EncryptKey(...) ](https://github.com/gallactic/gallactic/blob/b4ac9d41e87b2fe87ec7b06d9f23bb5a4e185713/keystore/key/encrypted.go#L160) function better to change the signature to `New (auth, lable string, ...)`
Please look here:

Reply:
I see what you mean. I'll make the changes in another commit

Reply:
Thanks @josefwijaya